nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/ConsumeAMQP.java

```diff
@@ -170,8 +208,8 @@ protected void processResource(final Connection connection, final AMQPConsumer c
 
             final BasicProperties amqpProperties = response.getProps();
             final Envelope envelope = response.getEnvelope();
-            final Map<String, String> attributes = buildAttributes(amqpProperties, envelope, context.getProperty(REMOVE_CURLY_BRACES).asBoolean(),
-                    context.getProperty(HEADER_SEPARATOR).toString());
+            final Map<String, String> attributes = buildAttributes(amqpProperties, envelope, context.getProperty(HEADER_FORMAT).toString(), context.getProperty(HEADER_KEY_PREFIX).toString(),
```

Comment:
It would be helpful to declare the values for `headerFormat` and `headerKeyPrefix`. In addition, `PropertyValue.getValue()` should be used instead of `PropertyValue.toString()`:
```suggestion
            final String headerFormat = context.getProperty(HEADER_FORMAT).getValue();
            final String headerKeyPrefix = context.getProperty(HEADER_KEY_PREFIX).getValue();
            final Map<String, String> attributes = buildAttributes(amqpProperties, envelope, headerFormat, headerKeyPrefix,
```